src/server_manager/web_app/digitalocean_account.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Account, RegionMap} from "../model/account";
+import {DigitalOceanSession, DropletInfo} from "../cloud/digitalocean_api";
+import {DigitalOceanServer, GetCityId} from "./digitalocean_server";
+import * as server from "../model/server";
+import * as crypto from "../infrastructure/crypto";
+import * as do_install_script from "../install_scripts/do_install_script";
+
+// Tag used to mark Shadowbox Droplets.
+const SHADOWBOX_TAG = 'shadowbox';
+const MACHINE_SIZE = 's-1vcpu-1gb';
+
+export class DigitalOceanAccount implements Account {
+  private servers: DigitalOceanServer[] = [];
+
+  constructor(
+      private digitalOcean: DigitalOceanSession, private image: string, private metricsUrl: string,
```

Comment:
I believe we can remove image, metricsURL and sentryApiUrl and inject the installScript instead.

Perhaps better on a separate PR